tools/meteor-services/deploy.js

```diff
@@ -71,8 +71,8 @@ const CAPABILITIES = ['showDeployMessages', 'canTransferAuthorization'];
 //   body, or a generic 'try again later' message, as appropriate
 
 function deployRpc(options) {
-  options = _.clone(options);
-  options.headers = _.clone(options.headers || {});
+  options = clone(options);
```

Comment:
Ah, right!  I had thought about this backwards and forgot that Underscore didn't have a deep clone; I've lapsed on my `_`!  Fixed & thanks!